db/pre-re/status.yml

```diff
@@ -1706,6 +1706,9 @@ Body:
     CalcFlags:
       Speed: true
       Aspd: true
+    States:
+      NoMove: true
+      NoMoveCond: true
```

Comment:
There is one more issue with Longing for Freedom. Now it enables movement correctly, but it doesn't allow normal attacks yet.

Reply:
Fixed in 948001e